cpp/net/connection_pool.cc

```diff
@@ -16,45 +17,57 @@ DEFINE_int32(url_fetcher_max_conn_per_host_port, 4,
 namespace cert_trans {
 namespace internal {
 
+ConnectionPool::Connection::Connection(evhtp_connection_t* conn,
+                                       HostPortPair&& other_end)
+    : conn_(CHECK_NOTNULL(conn)), other_end_(other_end) {
+}
+
+
+evhtp_connection_t* ConnectionPool::Connection::connection() const {
```

Comment:
Having this one in the header would let it inline where it's used, maybe a good idea?